src/pages/StateAndProps.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react'
```

Comment:
You don't need to import React.
```suggestion
import  { useState } from 'react'
```